backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4237,4 +4237,18 @@ databaseChangeLog:
             columnName: updated_by
         - addNotNullConstraint:
             tableName: device_specimen_type
-            columnName: is_deleted
\ No newline at end of file
+            columnName: is_deleted
+  - changeSet:
+      id: add-test-performed-code-column-to-device-supported-disease-table
+      author: boban@skylight.digital
+      comment: Add a test performed code column to store the specific LOINC code between a device and disease.
+      changes:
```

Comment:
does this need a rollback?

Reply:
Good point! Some places in this changelog that add new columns have the rollback, others do not 🤔. I just added since it makes sense to have it.

Reply:
I dont think you need to add a rollback since auto rollback is supported for this function. https://docs.liquibase.com/change-types/add-column.html#:~:text=PostgreSQL,Yes